### PR TITLE
Added comment entity.

### PR DESCRIPTION
**Pull request by VlasenkoYevhenii**

### CODE REVIEW

File: src/main/java/app/model/Comment.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "task_id", nullable = false)
+    private Task task;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
```
Comment (bodya4243) [inline src/main/java/app/model/Comment.java:31]: maybe for this has to be added @equalsHashcode and exlude this

File: src/main/java/app/model/Comment.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@Setter
+@Table(name = "comments")
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "task_id", nullable = false)
+    private Task task;
```
Comment (bodya4243) [inline src/main/java/app/model/Comment.java:27]: and here also

**Review by bodya4243 — COMMENTED** [review]
left two comments

File: src/main/java/app/model/Comment.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+@Entity
+@Getter
+@Setter
+@Table(name = "comments")
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
```
[the remark this answers is not in the document]
Comment (VlasenkoYevhenii) [reply src/main/java/app/model/Comment.java:26]: explicit is better than implicit, but ok)

[the remark this answers is not in the document]
Comment (makaroshyna) [reply src/main/java/app/model/Comment.java:26]: I think it will be okay not to remove it, we will need to check if this works in the future.

**Pull request closed without merging**